src/Common/src/System/Security/Cryptography/KeyBlobHelpers.cs

```diff
@@ -10,6 +10,45 @@ namespace System.Security.Cryptography
 {
     internal static class KeyBlobHelpers
     {
+        internal static byte[] ToUnsignedIntegerBytes(this ReadOnlyMemory<byte> memory, int length)
+        {
+            if (memory.Length == length)
+            {
+                return memory.ToArray();
+            }
+
+            ReadOnlySpan<byte> span = memory.Span;
+
+            if (memory.Length == length + 1)
```

Comment:
would it make sense to allow any number of prefixing zeros?

Reply:
No, the BER specification says that only one leading zero is permitted (and that when it's present the high bit of the next byte has to be set).  This is just removing the "sign padding" byte to convert from a BER INTEGER (signed BigInteger) to a crypto integer (unsigned BigInteger).